src/main/kotlin/no/nav/syfo/services/UpdateInfotrygdService.kt

```diff
@@ -600,10 +600,22 @@ class UpdateInfotrygdService {
     fun skalIkkeOppdatereInfotrygd(
         receivedSykmelding: ReceivedSykmelding,
         validationResult: ValidationResult
-    ): Boolean =
-        validationResult.ruleHits.isNotEmpty() && validationResult.ruleHits.any {
+    ): Boolean {
+
+        val partiallySickLeaveRule = validationResult.ruleHits.isNotEmpty() && validationResult.ruleHits.any {
```

Comment:
Kan vi vær så snill skrive sånne ting på norsk? Betyr dette gradert sykmelding?

Reply:
jeg stemmer for Engelsk :D

Reply:
Aner ikke. Dette lå her fra før av, og det var ikke så lett å tyde. Vi puttet det i en variabel for å gjøre det litt lettere å skjønne, men jeg er ikke sikker på hva regelen egentlig sier.

Den slår ut hvis ValidationRuleChain.PARTIALLY_COINCIDENT_SICK_LEAVE_PERIOD_WITH_PREVIOUSLY_REGISTERED_SICK_LEAVE og siste fom og siste tom har <= 3 dager mellom seg.

Tar gjerne imot forslag til bedre variabelnavn hvis du har :)

Reply:
Troller du, Andreas? Alltid vanskelig å vite ;) Joakim og Kevin som lagde disse reglene var enig i at det ikke var så lurt sånn i etterpåklokskapens lys å ha dem på engelsk, fordi det er domenespråk som er vanskelig å forstå når de oversettes siden de ikke har en entydig oversettelse :/ 

Jeg tror "PARTIALLY_COINCIDENT_SICK_LEAVE_PERIOD_WITH_PREVIOUSLY_REGISTERED_SICK_LEAVE" vil si at sykmeldingen er delvis overlappende med en sykmelding som allerede finnes i Infotrygd. Kanskje den kan hete `delvisOverlappendeSykmeldingRule`? Evt regel i stedet for rule, men jeg har ikke noe i mot at det heter rule siden rule/regel er ganske entydig og ikke NAV-spesifikt :)

Reply:
Takk for input, @tiafiring :)